src/STRPAK.CPP

```diff
@@ -1332,6 +1332,40 @@ BOOL strMatch(					// string match
 			bSeenDifSpace = TRUE;		// both space but different
 	}
 }			// strMatch
+//-----------------------------------------------------------------------------
+#if defined(__clang__) || defined(__GNUC__) && defined(__linux__)
```

Comment:
`#ifndef _MSC_VER`?

Reply:
I thought this was more explicit.

Reply:
What if the compiler is not MSVC, clang, or GCC?

Reply:
Because the `_str***` functions are specific to the Microsoft compiler, our redefine is relevant for (ALL != Microsoft), rather than for known specific compilers. I think I would use `#ifndef _MSC_VER` as well here. 
(But this is part of the complication of redefining a symbol with the same name as a proprietary one, which I personally _really_ don't like doing, but Chip's ok'ed.)